### PR TITLE
create YAML testing files for aero_model_wetdep

### PR DESCRIPTION
**Pull request by tangshuaiqi**

### CODE REVIEW

File: components/eam/src/chemistry/modal_aero/aero_model.F90 [hunk 1]
```diff
@@ -1654,7 +1666,7 @@ subroutine modal_aero_bcscavcoef_init
     ! set up temperature-pressure pair to compute impaction scavenging rates
     real(r8), parameter :: temp_0C = 273.16_r8        ! K
     real(r8), parameter :: press_750hPa = 0.75e6_r8   ! dynes/cm2
-#include "../yaml/aero_model/f90_yaml/modal_aero_bcscavcoef_init_beg_yml.f90"
+!#include "../yaml/aero_model/f90_yaml/modal_aero_bcscavcoef_init_beg_yml.f90"
```
Comment (singhbalwinder) [inline components/eam/src/chemistry/modal_aero/aero_model.F90]: I think these are commented out as you have generated data for these previously. You can uncomment these if you like.

**Review by singhbalwinder — APPROVED** [review]
Looks good! Left just one comments about commented out include files.

**Review by jeff-cohere — APPROVED** [review]

File: components/eam/src/chemistry/modal_aero/aero_model.F90 (hunk 1 above)
Comment (tangshuaiqi) [reply components/eam/src/chemistry/modal_aero/aero_model.F90]: Thank you for catching that! I will uncomment them.

**Comment by tangshuaiqi** [comment]
All tests passed. merge into maint

comp_yaml_aero_model_wetdep_mam4_org_v2_baselines: 9 tests
  PASS ERP_P64x1.ne4pg2_oQU480.F2010.compy_intel
  PASS ERS.ne4pg2_oQU480.F2010.compy_intel
  PASS PEM_P64x1.ne4pg2_oQU480.F2010.compy_intel
  PASS PET.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_D_Ln5_P1x1.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_D_Ln5_P32x1.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_D.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_Ly1.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS.ne30pg2_EC30to60E2r2.F2010.compy_intel

**Pull request merged**